tests/test_models.py

```diff
@@ -4012,40 +4019,6 @@ def test_formatted_date(self):
         unparsed = self.log.tz_date
         assert_equal(parsed, unparsed)
 
-    def test_resolve_node_same_as_self_node(self):
```

Comment:
Could you replace these tests with ones that check `log.node` vs `log.original_node` behavior for forks and registrations?